src/Setup/WixBA/Styles.xaml

```diff
@@ -1,103 +1,192 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
+<!--  Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.  -->
```

Comment:
I think VS is automatically formatting two spaces at the beginning and end
of any comment, even if I remove the space.  I was not aware of any
WixToolset style for XAML.  I also have XamlStyler installed, but this
reformatting is happening even without invoking it.

https://visualstudiogallery.msdn.microsoft.com/d6634d0e-38fb-48b6-829f-dadbc5c2fb62

I am aware that I could go off and create custom rules, but since custom
rules for wix do not appear to have been specified, I am just trying to
adapt to using the defaults regadless of my personal preferences.

On Mon, Aug 1, 2016 at 12:04 AM, Sean Hall <notifications@github.com> wrote:

> In src/Setup/WixBA/Styles.xaml
> <https://github.com/wixtoolset/wix4/pull/208#discussion_r72924491>:
>
> > @@ -1,103 +1,192 @@
> >  <?xml version="1.0" encoding="utf-8" ?>
> > -<!-- Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information. -->
> > +<!--  Copyright (c) .NET Foundation and contributors. All rights reserved. Licensed under the Microsoft Reciprocal License. See LICENSE.TXT file in the project root for full license information.  -->
>
> Extra spaces here, too.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/wixtoolset/wix4/pull/208/files/28721ab8e4985a5641b3efd7975a67cecd8f8d4a#r72924491>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHyCLevCqe-d3b8aSRFwbUITc4CYnvQXks5qbX5MgaJpZM4JVm8P>
> .
>